cmd/client/main.go

```diff
@@ -259,6 +261,10 @@ func (c *Client) run(o *GrpcProxyClientOptions) error {
 			transport := &http.Transport{
 				DialContext: dialer,
 			}
+			err = configureHTTP2Transport(transport)
```

Comment:
Can we wrap this in a CLI flag to control whether we have it turned on or not?
Otherwise looks good, thank you

Reply:
can do!

Reply:
done